src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -544,7 +545,14 @@ public Visitable visit(MethodCallExpr method, Void p) {
   @Override
   public Visitable visit(Parameter parameter, Void p) {
     try {
-      parameter.resolve().describeType();
+      if (parameter.getType() instanceof UnionType) {
+        UnionType unionType = parameter.getType().asUnionType();
+        for (var param : unionType.getElements()) {
+          param.resolve().describe();
```

Comment:
As you can see in the original codes, the purpose of this try block is to check if a parameter is unsolved. If it is, the parameter will be sent to the catch block where Specimin will create a synthetic class file for it. So I think we also need to do the samething with `param`. If it is unsolved (that is to say, an exception is thrown when we try to resolve it), `param` should be sent to the catch block so Specimin can create a synthetic class file for it.

Reply:
I have pushed a commit addressing this comment